### PR TITLE
ROC-5450 refactor dashboard

### PR DESCRIPTION
**Pull request by zakishaik**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5450

### Change description

 https://github.com/hmcts/cmc-citizen-frontend/pull/new/feature/ROC-5450-refactor-dashboard

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/test/features/dashboard/routes/index.ts [hunk 1]
```diff
@@ -265,6 +429,7 @@ describe('Dashboard page', () => {
               .get(Paths.dashboardPage.uri)
               .set('Cookie', `${cookieName}=ABC`)
               .expect(res => expect(res).to.be.successful.withText('Your money claims account', 'Make a new money claim'))
+              .expect(res => expect(res).to.be.successful.withText('<a href="/eligibility" class="newclaim" id="start-now" >Make a new money claim</a>'))
```
Comment (robertparkinson) [inline src/test/features/dashboard/routes/index.ts]: There's no need to check for the anchor link. the line above already checks for the text of the buttton

File: src/test/features/dashboard/routes/index.ts [hunk 2]
```diff
@@ -300,6 +465,7 @@ describe('Dashboard page', () => {
...
```
Comment (robertparkinson) [inline src/test/features/dashboard/routes/index.ts]: There's no need to check for the anchor link. the line above already checks for the text of the buttton

File: src/test/features/dashboard/routes/index.ts (hunk 1 above)
Comment (zakishaik) [reply src/test/features/dashboard/routes/index.ts]: removed this check

File: src/test/features/dashboard/routes/index.ts (hunk 2 above)
Comment (zakishaik) [reply src/test/features/dashboard/routes/index.ts]: removed this check

**Pull request merged**